### PR TITLE
fix: add akui prefix to common class names

### PR DESCRIPTION
**Pull request by RobChangCA**
# Pull Request Checklist

- [ ] Did you add new tests and confirm existing tests pass? (`yarn test`)
- [ ] Did you update relevant docs? (docs are found in the `site` folder, and guidelines for updating/adding docs can be found in the [contribution guide](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md))
- [ ] Do your commits follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] Does your PR title also follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) standard?
- [ ] If you have a breaking change, is it [correctly reflected in your commit message](https://www.conventionalcommits.org/en/v1.0.0/#examples)? (e.g. `feat!: breaking change`)
- [ ] Did you run lint (`yarn lint:check`) and fix any issues? (`yarn lint:write`)
- [ ] Did you follow the [contribution guidelines](https://github.com/alchemyplatform/aa-sdk/blob/main/CONTRIBUTING.md)?

# Description
We do not want to have our styles clobber the consumer's styles if they happen to use the same class names (btn, input, modal).  Added an akui prefix so that they will not clash.

# Testing steps
Go through the mint flow.  Buttons, inputs and modals should look the same.

---

## PR-Codex overview
This PR focuses on updating the styling of various components in the project by replacing the old class names with a new prefix (`akui-`). This enhances consistency across the UI components.

### Detailed summary
- Replaced `.modal` with `.akui-modal` in `modal.tsx`.
- Updated button classes from `.btn` to `.akui-btn` across multiple files.
- Modified form control classes from `.form-controls` to `.akui-form-controls`.
- Changed input classes from `.input` to `.akui-input`.
- Altered button variants to use `akui-` prefix in `button.tsx`.
- Updated styles in `globals.css` to reflect new class names.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: account-kit/react/src/tailwind/components/input.ts
Comment (moldy530) [inline account-kit/react/src/tailwind/components/input.ts:1]: line 35 in this file also applies styling to `input` as well, we should change that to `.akui-input` as well

File: account-kit/react/src/tailwind/components/modal.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import type { ComponentDef } from "../types";
 
 export const modalComponents: ComponentDef = {
-  ".modal": {
+  ".akui-modal": {
     "@apply radius-t-2 md:radius-2": {},
     "@apply bg-bg-surface-default": {},
-    ".modal-box": {
-      "@apply z-[1] px-6 py-4": {},
-    },
```
Comment (moldy530) [inline account-kit/react/src/tailwind/components/modal.ts:9]: is this not used?

File: account-kit/react/src/tailwind/components/input.ts
Comment (RobChangCA) [reply account-kit/react/src/tailwind/components/input.ts:1]: This is within the parent selector of .input which is used as a wrapper for our input components.  Do we want to select the input with a classname like .akui-input-field?

File: account-kit/react/src/tailwind/components/modal.ts (hunk 1 above)
Comment (RobChangCA) [reply account-kit/react/src/tailwind/components/modal.ts:9]: I was unable to find this classname in the codebase and when the modal renders in the dom.  I think it for something that was removed.

**Review by moldy530 — DISMISSED** [review]

**Pull request closed without merging**